### PR TITLE
 Device CRUD and new tooltips compatible with liveviews

### PR DESCRIPTION
**Pull request by coryjewell**
* Style index, edit, and new pages for devices
* Bootstrap tooltips are incompatible with liveviews so I removed them and replaced with a much lighter css-only approach

### CODE REVIEW

File: apps/nerves_hub_www/lib/nerves_hub_www_web/views/device_view.ex [hunk 1]
```diff
@@ -56,4 +56,12 @@ defmodule NervesHubWWWWeb.DeviceView do
...
 
   defdelegate device_status(device), to: Presence
+  
+  def take_device_tags(device, amount) do
+    Enum.take(device.tags, amount)
+  end
+  
+  def count_device_tags(device) do
+    Enum.count(device.tags)
+  end
```
Comment (coryjewell) [inline apps/nerves_hub_www/lib/nerves_hub_www_web/views/device_view.ex:66]: is it better to create methods this simple, or just use `Enum` in templates?

Comment (Mrjaco12) [reply apps/nerves_hub_www/lib/nerves_hub_www_web/views/device_view.ex:66]: I like the methods here for now. If we start to do a lot of really similar duplicate style methods then we can think about consolidation.

File: apps/nerves_hub_www/lib/nerves_hub_www_web/templates/device/index.html.leex [hunk 2]
```diff
@@ -24,56 +24,77 @@
...
         <td>
-          <a href="<%= Routes.device_path(@socket, :show, @org.name, @product.name, device.identifier) %>" class="">
+          <a href="<%= Routes.device_path(@socket, :show, @org.name, @product.name, device.identifier) %>" class="badge healthy-<%= device.healthy %>">
             <%= device.identifier %>
           </a>
         </td>
+
+        <td>
+          <div class="ml-3">
+          <%= if device.status == "offline" do %>
```
Comment (Mrjaco12) [inline apps/nerves_hub_www/lib/nerves_hub_www_web/templates/device/index.html.leex:44]: Do we know what all of the possible statuses are? I wonder if there is a possibility that the device might not be online but have a different status like "error" or something.

**Review by Mrjaco12 — APPROVED** [review]
Looks good from here. Tests and styles all good?

**Pull request merged**